frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -63,6 +63,7 @@
 import java.io.IOException;
 import java.io.PrintWriter;
 import java.util.*;
+import java.util.regex.Pattern;
```

Comment:
This import looks unnecessary

Reply:
yes its unnecessary. I removed this import